custom_components/rpi_rf/switch.py

```diff
@@ -1,5 +1,7 @@
 """Support for a switch using a 433MHz module via GPIO on a Raspberry Pi."""
 from __future__ import annotations
+from email.policy import default
+from enum import unique
```

Comment:
not needed?
```suggestion
```

Reply:
Sorry, didn't realise that my linter autoimported this when I used the unique variable name